Lib/fontTools/varLib/__init__.py

```diff
@@ -711,6 +713,28 @@ def _add_CFF2(varFont, model, master_fonts):
 	merge_region_fonts(varFont, model, ordered_fonts_list, glyphOrder)
 
 
+def _add_COLR(font, model, master_fonts, axisTags):
+	merger = COLRVariationMerger(model, axisTags, font)
+	merger.mergeTables(font, master_fonts)
+	store = merger.store_builder.finish()
+
+	colr = font["COLR"].table
+	if store.VarData:
+		mapping = store.optimize()
+		colr.VarStore = store
+		# the special 0xFFFF/0xFFFF (for no variations) always maps to itself
+		mapping[ot.NO_VARIATION_INDEX] = ot.NO_VARIATION_INDEX
```

Comment:
This needs to go into the VarStore logic itself.
Generally this issue: https://github.com/fonttools/fonttools/issues/2211

Reply:
Ie. Other places are affected as well.